src/imex.rs

```diff
@@ -22,8 +22,9 @@ use crate::message::{Message, MsgId};
 use crate::mimeparser::SystemMessage;
 use crate::param::*;
 use crate::pgp;
+use crate::rusqlite::types::ValueRef;
 use crate::sql::{self, Sql};
-use crate::stock::StockMessage;
+use crate::stock::StockMessage; //cs
```

Comment:
Drop these comments, especially in the line which is not modified by this PR.

Reply:
must be a mistake :-)